doc-templates/README.template.md

```diff
@@ -161,7 +161,7 @@ cluster in the given cloud provider
 
 ### Specifications
 
-* [Hints.yaml format](./docs/hints-yaml-format.md)
+* [Scenario.yaml format](./docs/scenario-yaml-format.md)
```

Comment:
link does not work in file

Reply:
that's because this is the template, the generated README is in the root so the links will work.  I'll be changing it to `tasks-yaml-format.md` though